cmd/wallet-web/src/store/modules/user.js

```diff
@@ -218,11 +219,15 @@ export default {
             'db-namespace': rootState.user.username,
           });
 
-          let agent = await new Agent.Framework(opts);
-          commit('setInstance', { instance: agent, user: rootState.user.username });
-          // TODO to be moved from here to 'loadOIDCUser' in case server based universal wallet.
-          await dispatch('unlockWallet');
-          commit('startAllNotifiers');
+          try {
+            let agent = await new Agent.Framework(opts);
+            commit('setInstance', { instance: agent, user: rootState.user.username });
+            // TODO to be moved from here to 'loadOIDCUser' in case server based universal wallet.
+            await dispatch('unlockWallet');
+            commit('startAllNotifiers');
+          } catch (e) {
+            console.error(e);
```

Comment:
is it left over? if not then add a prefix to error message so that it will be easy to locate the origin of error

Reply:
It is left on purpose, the browser console would print the filename and the line number if this throws